libmscore/layout.cpp

```diff
@@ -764,26 +764,30 @@ static qreal layoutAccidental(AcEl* me, AcEl* above, AcEl* below, qreal colOffse
       qreal lx = colOffset;
       Accidental* acc = me->note->accidental();
       qreal mag = acc->mag();
+      const qreal ledgerAdjust = -(acc->staff()->spatium(acc->tick()) * acc->score()->styleS(Sid::ledgerLineLength).val());
```

Comment:
this will now be calculated by every single accidental. Could that be calculated only if needed?

Reply:
Sure, with some effort.  But maybe not worth it.  Normally we don't need to layout out every single accidental in the whole score on every layout (unlike the case when this code was written).  We do far more expensive calculations regularly.

Reply:
One quick check would be, only do the calculation is the chord actually has ledger lines - that much should be simple to determine up front.